docs/self-managed/operational-guides/backup-restore/backup-and-restore.md

```diff
@@ -38,8 +38,8 @@ To back up a Camunda 8 cluster, execute the following sequential steps:
 4. Wait until the backup `x` of Optimize is complete. See [how to monitor an Optimize backup](/self-managed/operational-guides/backup-restore/optimize-backup.md).
 5. Wait until the backup `x` of Operate is complete. See [how to monitor an Operate backup](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md).
 6. Wait until the backup `x` of Tasklist is complete. See [how to monitor a Tasklist backup](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md).
-7. Pause exporting in Zeebe. See [Zeebe management API](/self-managed/zeebe-deployment/operations/management-api.md).
-8. Take a backup `x` of the exported Zeebe records in Elasticsearch using the Elasticsearch Snapshots API.
+7. Soft pause exporting in Zeebe. See [Zeebe management API](/self-managed/zeebe-deployment/operations/management-api.md).
```

Comment:
:wrench: If possible link directly to the section of "soft pause".

Reply:
In the other examples, they seem to just link to page. In the older versions where it is possible to link should we still do it? Seems out of place 🤔

Reply:
No strong opinion. But I think it is easy to miss that it is "soft" pause. So if you link directly to it, it reduces the chance of confusion.